workday/12.1/modules/ROOT/pages/workday-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,271 @@
+= Workday - Upgrade Workday Connector - Mule 4
+:page-aliases: connectors::workday/workday-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for Workday (Workday Connector) to version 12.x.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|11.1.x |12.x.x
+|===
+
+== What's Changed
+
+=== 12.1
+
+This release contains the following changes:
+
+=== 12.0.0
+
+This release contains the following changes:
+
+* Upgraded the Workday API to 34.0
+* Added the `Student Core` operation, which enables you to invoke the Workday Student Core Web Service
+* Replaced the `On New Objects` input source with the `On Modified Objects` input source, which polls data more efficiently.
++
+For more information, see <<on-modified-objects,On Modified Objects Input Source>>.
++
+* Changed all operations so that they return headers as attributes
+* Added the ability to create response timeouts for the input source (`On Modified Objects`) and all connector operations by using the new `Response Timeout` and `Response Timeout Unit` fields
+* Split the connection timeout fields into value and time unit fields
++
+For more information, see <<changes-in-connection-types,Changes in Connection Types>>.
++
+* Removed the `blocking` field from the `Reconnect` and `Reconnect Forever` objects
+* Removed the `WORKDAY:UNKNOWN` error type
+
+[[changes-in-connection-types]]
+== Changes in Connection Types
```

Comment:
This is a peer topic to What's Changed?

Reply:
No, it's a child section of parent What's Changed